experiments/integrated/fluxnet/fluxnet_simulation.jl

```diff
@@ -0,0 +1,12 @@
+t0 = FT(120 * 3600 * 24)# start mid year
+N_spinup_days = 10
+N_days = N_spinup_days + 30
+dt = FT(120)
```

Comment:
dt for sure, and possibly (t0, tf) are site specific. But the rest of this (algorithm, N_spinup, timestepper, etc) should be generic between sites.

Reply:
I was intending this file to be generic since it lives outside of the sub-directories for each of the sites. But I think it's a good idea to have a simulation parameters file for each experiment in their directories giving dt as well as the domain variables that change site by site.